changelogs/fragments/248-token_validate-change-default.yml

```diff
@@ -0,0 +1,3 @@
+---
+deprecated_features:
+  - token_validate options - the shared auth option ``token_validate`` will change its default from ``True`` to ``False`` in community.hashi_vault version 4.0.0. The ``vault_login`` lookup and module will keep the default value of ``True`` (https://github.com/ansible-collections/community.hashi_vault/issues/248).
```

Comment:
In Ansible docs, we usually use lower-case names for the booleans. But if you use the upper-case version everywhere, feel free to keep this as-is.